lib/Optimizer/Transforms/QuakeToQTXConverter.h

```diff
@@ -92,10 +92,10 @@ class ConvertToQTXRewriter final : public mlir::PatternRewriter {
 
   /// Hook for inserting operations, and make sure that newly inserted ops are
   /// tracked.
-  void notifyOperationInserted(mlir::Operation *op) override;
+  void notifyOperationInserted(mlir::Operation *op) /*override*/;
```

Comment:
@boschmitt Looks like there is a new Listener class for this. That these vanished may explain the test failure.